src/main/kotlin/com/noumenadigital/npl/cli/commands/registry/TestCommand.kt

```diff
@@ -77,6 +77,15 @@ data class TestCommand(
             }
 
             coverageAnalyzer.writeSummary(output::info)
+
+            if (showCoverage && File.separatorChar == '\\') {
+                val coverageFile = File(".").canonicalFile.resolve(COVERAGE_FILE)
+                if (coverageFile.exists()) {
+                    val fixed = coverageFile.readText().replace('\\', '/')
```

Comment:
and here we're doing the opposite, but the consumer of this is tooling that expect unix slashes I believe